autobump-maven/src/main/java/com/github/autobump/maven/model/MavenDependencyBumper.java

```diff
@@ -0,0 +1,62 @@
+package com.github.autobump.maven.model;
+
+import com.github.autobump.core.exceptions.DependencyParserException;
+import com.github.autobump.core.model.Bump;
+import com.github.autobump.core.model.Dependency;
+import com.github.autobump.core.model.DependencyBumper;
+import com.github.autobump.core.model.Workspace;
+import org.apache.maven.model.InputLocation;
+import org.apache.maven.model.InputSource;
+import org.apache.maven.model.io.xpp3.MavenXpp3ReaderEx;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
+public class MavenDependencyBumper implements DependencyBumper {
+
+    @Override
+    public void bump(Workspace workspace, Bump bump) {
+        try (Reader reader = workspace.getDependencyDocument(MavenDependencyResolver.DEPENDENCY_FILENAME)) {
+            InputLocation versionLocation = findVersionLine(reader, bump.getDependency(), workspace.getProjectRoot());
+            updateDependency(versionLocation, bump);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
```

Comment:
The IOEXception gets thrown in methods updateDependency and findVersionLine

it's possible to force those methods to throw that exception through inheritance